app/controllers/admin/tenants_controller.rb

```diff
@@ -32,4 +33,13 @@ def update
     def tenant_params
       params.require(:tenant).permit(:name, :schema, :schema_type)
     end
+
+    def update_default_tenant_administrator
+      Tenant.switch(@tenant.schema) do
+        default_admin_account = User.administrators.last
```

Comment:
Although it doesn't really matter, using `first` IMHO expresses a bit better the fact that we're changing the default administrator :thinking:.

```suggestion
        default_admin_account = User.administrators.first
```